app/src/main/res/values-ru/strings.xml

```diff
@@ -0,0 +1,218 @@
+        Посетители ежегодного конгресса и лагеря, организованного Chaos Computer Club
+        используют приложение для отслеживания программы мероприятий.
+    </string>
+
+    <!-- SnackEngage -->
+    <string name="snack_engage_rate_title">Нравится приложение?</string>
+    <string name="snack_engage_rate_action">Оценить</string>
+    <string name="snack_engage_google_play_beta_testing_title">Любишь всё новое?</string>
+    <string name="snack_engage_google_play_beta_testing_action">Стать бета-тестером</string>
+
+    <string name="snack_engage_c3nav_url" translatable="false">
+        https://play.google.com/store/apps/details?id=de.c3nav.droid
+    </string>
+    <string name="snack_engage_c3nav_unique_id" translatable="false">c3nav</string>
+    <string name="snack_engage_c3nav_title">Навигация внутри 35C3! с приложением c3nav</string>
+    <string name="snack_engage_c3nav_action">Скачать сейчас!</string>
+
+    <!-- TraceDroid -->
+    <string name="trace_droid_dialog_title">
+        Отправить отчёт об ошибке <xliff:g id="appName" example="33C3 Schedule">%s</xliff:g> &#8230;
+    </string>
+    <string name="trace_droid_dialog_message">
+        Обнаружена проблема с приложением <xliff:g id="appName" example="33C3 Schedule">%s</xliff:g>.
+        Было бы полезно, если бы вы смогли предоставить отчет об ошибке,
+        чтобы можно было определить источник проблемы.
+    </string>
+    <string name="trace_droid_button_title_send">
+        Отправить E-mail
+    </string>
+    <string name="trace_droid_no_email_app">
+        Не установлено приложение для отправки E-mail.
+    </string>
+    <string name="trace_droid_button_title_no">
+        Нет
+    </string>
+    <string name="trace_droid_button_title_later">
+        Позже
+    </string>
+    <string name="trace_droid_app_chooser_title">
+        Открытый черновик с &#8230;
+    </string>
+
+    <!-- Notifications -->
+    <string name="notifications_event_alarm_channel_description">
+        Пользовательские напоминания для различных мероприятий.
+    </string>
+    <string name="notifications_event_alarm_channel_name">Напоминания для мероприятий</string>
+    <string name="notifications_event_alarm_content_text">Напоминание для мероприятия</string>
+    <string name="notifications_event_alarm_content_title" translatable="false">@string/app_name</string>
+    <string name="notifications_schedule_update_channel_description">
+        Автоматическое обновление расписания можно выбрать в настройках приложения.
+    </string>
+    <string name="notifications_schedule_update_channel_name">Обновление расписания</string>
+    <plurals name="notifications_schedule_changes">
+        <item quantity="one"><xliff:g id="count" example="1">%d</xliff:g> лекция изменена.</item>
+        <item quantity="few"><xliff:g id="count" example="2">%d</xliff:g> лекции изменены.</item>
+        <item quantity="other"><xliff:g id="count" example="3">%d</xliff:g> лекций изменено.</item>
+    </plurals>
```

Comment:
The lint check in Android Studio tells that for `notifications_schedule_changes` also `many` should be defined. Can you tell if the warning is correct and if we need the another item here because of a special plural word?